addons/mass_mailing_sms/wizard/sms_composer.py

```diff
@@ -24,6 +24,10 @@ def _get_unsubscribe_url(self, res_id, trace_code, number):
             '/sms/%s/%s' % (self.mailing_id.id, trace_code)
         )
 
+    @api.model
+    def _get_unsubscribe_info(self, url):
+        return _('STOP SMS : %s', url)
```

Comment:
Maybe ``_('STOP SMS : %(stop_url)s')`` ? Would give a bit of context for translation ...